front/php/server/db.php

```diff
@@ -51,6 +51,7 @@ function OpenDB () {
   }
 
   $db = SQLite3_connect(true);
+  $db->exec('PRAGMA journal_mode = wal;');
```

Comment:
Cool, so this seems to help, yes? Nice, good to know

Reply:
Seems to. Found that after a bit of searching around